src/components/form/fields/schema-field.ts

```diff
@@ -260,4 +268,30 @@ export class SchemaField extends React.Component<FieldProps> {
 
         return React.createElement(JSONSchemaField, fieldProps);
     }
+
+    /**
+     * Gets the path to the current property within the schema
+     *
+     * @param {string} schemaId the id of the schema
+     * @returns {string[]} an array with the schema path for the current property
+     * @example
+     * const schemaId = 'root_sections_0_controls_0_name';
+     * const schemaPath = getSchemaPath(schemaId);
+     * // => ['sections', '0', 'controls', '0', 'name']
+     */
+    private getSchemaPath(schemaId: string): string[] {
+        const paramString = window.location.href.split('?')[1];
+        const selectedConfigPath = new URLSearchParams(paramString).get(
+            'selectedConfig'
+        );
+
+        if (!selectedConfigPath) {
+            return [];
+        }
```

Comment:
We can't couple the web client with lime elements like this. Can we get the full path in some other way?

Reply:
We can maybe look in `this.formInfo.rootValue` inside the component instead?

Reply:
Ah yeah I really don't know why I did it like this here since I was thinking about that when implementing it from the start, sorry. Too much context switching after the first time which was a long time ago :) I'll look at this again!

Reply:
> We can maybe look in `this.formInfo.rootValue` inside the component instead of `this.config`?

Excellent idea!